layouts/partials/feedback.html

```diff
@@ -0,0 +1,28 @@
+<div class="govuk-grid-row  govuk-width-container dfe-width-container">
+  <div class="govuk-grid-column-one-third">
+    <div class="dfe-icon-container">
```

Comment:
I think this class is adding `text-align: center` to the title, body and link below. The prototype has everything left aligned underneath the icon.

Actually - could tackle this and comment below in  a separate ticket as it's not related to this page!